keyboards/handwired/wwa/rosetta/info.json

```diff
@@ -0,0 +1,56 @@
+{
+    "manufacturer": "spacehangover",
+    "keyboard_name": "Rosetta",
+    "maintainer": "spacehangover",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "midi": true
+    },
+    "build":{
+        "lto": true
+    },
+    "matrix_pins": {
+        "cols": ["C6", "D4", "D0", "D1"],
+        "rows": ["B4", "E6", "D7"]
+    },
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0006",
+        "vid": "0x5757"
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
```

Comment:
Given that neither of the provided keymaps include modifier keys, this functionality is unused.
```suggestion
```